lib/rest/lib/params.c

```diff
@@ -26,7 +25,8 @@ get_value(const struct rest_params *const params, const char *const key)
     if (!params)
         return NULL;
 
-    TAILQ_FOREACH(param, (struct evkeyvalq *)params, next) {
+    TAILQ_FOREACH(param, (struct evkeyvalq *)params, next)
+    {
```

Comment:
Why didn't it put a space after TAILQ_FOREACH like it does the list macros?  And why did it drop the open brace down a line?

Reply:
Maybe it think it's a local function definition?  I will add to list of `foreach` macros.